_includes/releases/release-downloads-docker-image.md

```diff
@@ -32,6 +37,12 @@ This release was withdrawn, and we've removed the links to the downloads and Doc
 <h4>SQL-only command-line client executable</h4>
 
 <div><div id="os-tabs" class="filters clearfix">
+{% if release.linux_arm == "false" %}
+    <a href="https://binaries.cockroachdb.com/cockroach-sql-{{ release.version }}.linux-amd64.tgz"><button id="linux" class="filter-button" data-scope="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
```

Comment:
The filename format is similarly different for the cockroach-sql files. Here is the alpha.4 one: `cockroach-sql-v22.2.0-alpha.4.linux-3.7.10-gnu-aarch64.tgz`